embc-app/Models/Db/IncidentTaskEvacueeSummary.cs

```diff
@@ -0,0 +1,19 @@
+using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations.Schema;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Gov.Jag.Embc.Public.Models.Db
+{
+    public class IncidentTaskEvacueeSummary
```

Comment:
The idea of adding a class to accommodate a view looks insane, is this the only way of doing it in EF?